_emulator/.firebaserc

```diff
@@ -1,5 +1,5 @@
 {
   "projects": {
-    "default": "demo-test"
+    "default": "dev-extensions-testing"
```

Comment:
Prefixed `demo-` naming ensure no live services are accidentally called. Would this be the correct naming for this type of update?

Source: https://firebase.google.com/docs/emulator-suite/connect_firestore

Reply:
This was meant for getting the API key (for e2e tests) from cloud secrets, but I'm looking into a way to revert back to the demo project and set keys as secrets locally in CI.